src/dropin-model.js

```diff
@@ -98,6 +99,10 @@ DropinModel.prototype.confirmDropinReady = function () {
   this._setupComplete = true;
 };
 
+DropinModel.prototype.verifyCardReady = function () {
+  this._shouldWaitForVerifyCard = !this._shouldWaitForVerifyCard;
```

Comment:
maybe I need caffeine, but do I undertand this evaluation correctly?

we are setting the value of `_shouldWAitForVerifyCard` to the opposite of `_shouldWaitForVerifyCard`? 

So false?

Reply:
Is this some JS magic that I'm simply forgetting lol

Reply:
It's whatever the opposite value used to be (toggling between true/false)

Reply:
Oscar is correct in this was meant to toggle between true/false, however based on discussion we are removing this setter method and instead modifying the `shouldWaitForVerifyCard` property directly